tanglescope/common/iri.hpp

```diff
@@ -174,6 +174,22 @@ class LMHSMessage : public IRIMessage {
   std::string _latestSolidMilestoneHash;
 };
 
+class LMSIMessage : public IRIMessage {
+ public:
```

Comment:
you might want to delete the default constructor?

Reply:
Defining a one parameter constructor will prevent the compiler from generating default one,
so no unintentional usage of that is possible